Gifski/Intents.swift

```diff
@@ -86,29 +316,44 @@ struct ConvertIntent: AppIntent, ProgressReportingIntent {
 	)
 	var dimensionsHeight: Int?
 
+	@Parameter(
+		title: "Crop",
+		description: "Optionally crop the video.",
+	)
+	var crop: CropEntity?
+
+	@Parameter(
+		title: "Output Mode",
+		description: "Output an animated GIF or a single frame preview of the GIF.",
+		default: .animatedGIF
+	)
+	var outputMode: ConvertOutputMode
```

Comment:
Since it's just two cases, would be better to make it just a bool. Like `isPreview`.